tracetool/README.md

```diff
@@ -0,0 +1,70 @@
+# Linux trace tool
+
+
+## Introduction
+
+This is python script tool to analysis Linux ftrace, support trace events from
+Linux kernel(e.g. i915) and from user mode lib/application(e.g. libva, media driver).
+
+
+## Linux trace capture
+
+1. Install trace-cmd::
```

Comment:
Here and below, remove 2nd `:` (likely borrowed from some rst document).

Reply:
fixed